models.go

```diff
@@ -55,6 +55,43 @@ var nip47MethodIcons = map[string]string{
 	NIP_47_LIST_TRANSACTIONS_METHOD: "transactions",
 }
 
+type InfoResponse struct {
+	User        User   `json:"user"`
+	BackendType string `json:"backendType"`
+	Csrf        string `json:"csrf"`
+}
+
+type CSRFResponse struct {
+	Csrf string `json:"csrf"`
+}
+
+type ShowAppResponse struct {
+	App                   App           `json:"app"`
+	BudgetUsage           int64         `json:"budgetUsage"`
+	Csrf                  string        `json:"csrf"`
```

Comment:
it seems strange to include the Csrf in all models. Let's put it in one place if we are going to use it